neurovault/api/serializers.py

```diff
@@ -229,8 +230,7 @@ class Meta:
 class CollectionSerializer(serializers.ModelSerializer):
     url = HyperlinkedImageURL(source='get_absolute_url', read_only=True)
     owner = serializers.ReadOnlyField(source='owner.id')
-    images = ImageSerializer(many=True, source='image_set')
-    nidm_results = NIDMResultsSerializer(many=True, source='nidmresults_set')
+    images = ImageSerializer(many=True, source='basecollectionitem_set')
```

Comment:
Does it mean that `images` attribute will contain statisticmaps, nidmresults and atlases?